assets/templates/app.hbs

```diff
@@ -0,0 +1,9 @@
+{{!-- stuff goes here --}}
+<div class="full">
+	<div class="area">
+		<textarea>Join the discussion...</textarea>
```

Comment:
Instead of showing an initial value of "Join the discussion" use a `placeholder` attribute. Placeholders allow you to click and have an empty textarea or input to work with while setting the value requires the user to delete the value.